waspc/data/Generator/templates/server/src/actions/_action.js

```diff
@@ -6,11 +6,13 @@ import prisma from '../dbClient.js'
 {=! TODO: This template is exactly the same at the moment as one for queries,
           consider in the future if it is worth removing this duplication. =}
 
-export default async (args, context) => {
-  context = { ...context, entities: {
-    {=# entities =}
-    {= name =}: prisma.{= prismaIdentifier =},
-    {=/ entities =}
-  }}
-  return {= jsFnIdentifier =}(args, context)
+export default async function (args, context) {
+  return {= jsFnIdentifier =}(args, {
+    ...context,
+    entities: {
+      {=# entities =}
+      {= name =}: prisma.{= prismaIdentifier =},
+      {=/ entities =}
+    },
+  })
```

Comment:
This function is exactly the same as the one in `src/queries/_query.ts`, but I think it's better to keep them separate  because they're semantically different despite having the same structure at the moment.

Reply:
Yup, I think that is correct approach! We have that happening with queries and actions on many places in the code and I think that is ok for now.